src/internal_events/aws_ecs_metrics.rs

```diff
@@ -13,13 +13,17 @@ pub struct AwsEcsMetricsReceived {
 
 impl InternalEvent for AwsEcsMetricsReceived {
```

Comment:
When a URL is relevant to a component, an `endpoint` should be included in the event per https://github.com/vectordotdev/vector/blob/master/docs/specs/component.md#options.

I can see this is already the case with `AwsEcsMetricsParseError`, but I think other events should include it too.